ansible/roles/ssc/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+# SSC Configuration
+ssc_enabled: false
+ssc_version: "0.5.0-alpha.2"
+# Hardcoded values are now used directly in templates
+
+# SSC Resources
+ssc_resources:
+  requests:
+    memory: "512Mi"
+    cpu: "0.2"
+  limits:
+    memory: "1Gi"
+    cpu: "0.5"
+
```

Comment:
There is one important thing we need here to be future proff:
```
ssc_mode: {{ mode }}
```

Basically, we will have validators running an ssc full node, while having the validator outside the cluster. With this config, the mode defaults to the global one, but they can override.